apps/site/open-next.config.ts

```diff
@@ -0,0 +1,6 @@
+import { defineCloudflareConfig } from '@opennextjs/cloudflare';
+import kvIncrementalCache from '@opennextjs/cloudflare/overrides/incremental-cache/kv-incremental-cache';
+
+export default defineCloudflareConfig({
+  incrementalCache: kvIncrementalCache,
+});
```

Comment:
```suggestion
import { defineCloudflareConfig } from '@opennextjs/cloudflare';
import incrementalCache from '@opennextjs/cloudflare/overrides/incremental-cache/kv-incremental-cache';

export default defineCloudflareConfig({ incrementalCache });
```

Reply:
this actually reminds me that the Cloudflare/OpenNext team is now recommending the R2 incremental cache solution, I should probably update this to use that instead of the KV one 👍